packages/fxa-auth-server/test/local/routes/account.js

```diff
@@ -3396,95 +3420,205 @@ describe('/account', () => {
+      } catch (err) {
+        failed = true;
+        assert.equal(err.errno, error.ERRNO.UNEXPECTED_ERROR);
+      }
+
+      assert.isTrue(failed);
+    });
+
+    it('should not return stripe.customer result when subscriptions are disabled', () => {
+      return runTest(buildRoute(false), request, (result) => {
+        assert.deepEqual(result, {
+          subscriptions: [],
+        });
+
+        assert.equal(log.begin.callCount, 1);
+        assert.equal(mockStripeHelper.customer.callCount, 0);
       });
-      assert.equal(log.begin.callCount, 1);
-      assert.equal(mockStripeHelper.customer.callCount, 1);
-      assert.equal(mockStripeHelper.subscriptionsToResponse.callCount, 0);
     });
   });
 
-  it('should propagate other errors from stripe.customer', async () => {
-    mockStripeHelper.customer = sinon.spy(() => {
-      throw error.unexpectedError();
+  describe('Google Play subscriptions', () => {
+    const mockIapSubscription = {
+      auto_renewing: true,
+      expiry_time_millis: Date.now(),
+      package_name: 'org.mozilla.cooking.with.foxkeh',
+      sku: 'org.mozilla.foxkeh.yearly',
+      start_time_millis: Date.now(),
+      _subscription_type: MozillaSubscriptionTypes.IAP_GOOGLE,
+      product_id: 'iap_prod_lol',
+      product_name: 'LOL daily',
+    };
+
+    const mockIapSubscriptionResponse = [
+      {
+        created: Math.round(mockIapSubscription.start_time_millis / 1000),
+        product_id: mockIapSubscription.product_id,
+        product_name: mockIapSubscription.product_name,
+      },
+    ];
+
+    beforeEach(async () => {
+      mockCustomer = undefined;
+      mockSubscriptionsResponse = [];
+      mockStripeHelper = mocks.mockStripeHelper([
+        'customer',
+        'subscriptionsToResponse',
+        'iapSubscriptionsToResponse',
+      ]);
+      mockStripeHelper.customer = sinon.spy(async (uid, email) => mockCustomer);
+      mockStripeHelper.subscriptionsToResponse = sinon.spy(
+        async (subscriptions) => mockSubscriptionsResponse
+      );
+      mockStripeHelper.iapSubscriptionsToResponse = sinon.spy(
+        async (subscription) => mockIapSubscriptionResponse
+      );
+      Container.set(CapabilityService, sinon.fake);
+      mockPlaySubscriptions = mocks.mockPlaySubscriptions([
+        'getActiveGooglePlaySubscriptions',
+      ]);
+      Container.set(PlaySubscriptions, mockPlaySubscriptions);
+      mockPlaySubscriptions.getActiveGooglePlaySubscriptions = sinon.spy(
+        async (uid) => [mockIapSubscription]
+      );
     });
 
-    let failed = false;
-    try {
-      await runTest(buildRoute(), request, () => {});
-    } catch (err) {
-      failed = true;
-      assert.equal(err.errno, error.ERRNO.UNEXPECTED_ERROR);
-    }
+    it('should return formatted Google Play subscriptions when subscriptions are enabled', () => {
+      return runTest(buildRoute(), request, (result) => {
+        assert.deepEqual(mockStripeHelper.customer.args[0], [{ uid, email }]);
```

Comment:
Why do we have to wrap the `args` assertion expected values in an array? I see that was done before my additional tests for e.g. `assert.deepEqual(mockStripeHelper.customer.args[0], [{ uid, email }]);`.